run.py

```diff
@@ -30,6 +32,11 @@ def create_app(config):
     return app
 
 
+def retry_if_database_error(exception):
+    return isinstance(exception, DatabaseError)
+
+
+@retry(retry_on_exception=retry_if_database_error, wait_fixed=2000, stop_max_delay=30000)
```

Comment:
Do you want to gracefully handle the `RetryError` that is raised?

Reply:
https://github.com/rholder/retrying/blob/1d5699348d707e377aad7488da6a8a1b48a65933/retrying.py#L286

Reply:
I guess the only way we can handle it is by logging the error as at that point it seems irrecoverable?